.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
According to the [project requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md) you should add a `test.md` file to your `.gitignore` file. Kindly fix this.

<img width="732" alt="Screen Shot 2022-05-24 at 8 00 22" src="https://user-images.githubusercontent.com/82426682/170040781-2b7389bf-1090-4e83-bd76-e304fca97995.png">
.